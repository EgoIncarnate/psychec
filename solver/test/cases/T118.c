void f() {
    switch (a) {
    case b:
        return;
    case c:
        return;
    case 128:
        return;
    case d:
        return;
    default:
        return;
    }
}
