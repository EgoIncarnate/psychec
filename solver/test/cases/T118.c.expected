uintptr_t a;
#define b 131
#define c 130
#define d 129
